proto/directions_options.proto

```diff
@@ -6,8 +6,14 @@ message DirectionsOptions {
     kKilometers = 0;
     kMiles = 1;
   }
+  
+  enum Format {
+    json = 0;
```

Comment:
should these be like others? kJson

Reply:
talked about this offline, but basically these strings are part of the request API and it seemed less confusing to just have `json` or `gpx` in the actually external web API than with the leading `k`